CodePush.m

```diff
@@ -100,16 +109,13 @@ + (void)setUsingTestConfiguration:(BOOL)shouldUseTestConfiguration
 }
 
 /*
- * This is used to clean up all test updates. It can only be used
- * when the testConfigurationFlag is set to YES, otherwise it will
- * simply no-op.
+ * WARNING: This cleans up all downloaded and pending updates.
  */
-+ (void)clearTestUpdates
++ (void)clearUpdates
 {
-    if ([CodePush isUsingTestConfiguration]) {
-        [CodePushPackage clearTestUpdates];
-        [self removePendingUpdate];
-    }
+    [CodePushPackage clearUpdates];
```

Comment:
Should we not do this part in debug mode? Otherwise, you could continuously get the same CodePush update over and over while deploying new binaries to the simulator and device right?